src/pip-delete-this-directory.txt

```diff
@@ -0,0 +1,5 @@
+This file is placed here by pip to indicate the source was put
```

Comment:
Sure, not sure how that ended up in there

Regards,
Clinton Carpene

From:  Sander <notifications@github.com>
Reply-To:  OpenPoGo/OpenPoGoBot
<reply+002bb0ec2b794e58818cd631c052917915747e17fa365cff92cf0000000113b2d0ac9
2a170ce045628ae@reply.github.com>
Date:  Friday, 29 July 2016 at 4:07 PM
To:  OpenPoGo/OpenPoGoBot <OpenPoGoBot@noreply.github.com>
Cc:  Clinton Carpene <clinton.carpene@gmail.com>, Author
<author@noreply.github.com>
Subject:  Re: [OpenPoGo/OpenPoGoBot] Config options for Webserver and
softban handling (#110)


In src/pip-delete-this-directory.txt
<https://github.com/OpenPoGo/OpenPoGoBot/pull/110#discussion_r72755374> :
> @@ -0,0 +1,5 @@
> +This file is placed here by pip to indicate the source was put
Can you remove this file?


You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/OpenPoGo/OpenPoGoBot/pull/110/files/25ae2270ee9f16b291a9
22064f7f1f3c435c731b#r72755374> , or mute the thread
<https://github.com/notifications/unsubscribe-auth/ACuw7IqAThIyYfHbpdKUkuIWh
X-Zt0Ruks5qabSsgaJpZM4JX9gY> .